addons/l10n_din5008/report/din5008_report.xml

```diff
@@ -72,19 +72,20 @@
                                         <span>|</span> <span t-field="company.country_id.name"/>
                                     </t>
                                     <hr class="company_invoice_line" />
-                                    <div t-if="address">
+                                    <span t-if="address">
                                         <t t-out="address"/>
-                                    </div>
-                                    <div t-else="fallback_address">
+                                    </span>
+                                    <span t-else="fallback_address">
                                         <t t-out="fallback_address"
                                            t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}' />
-                                    </div>
+                                    </span>
                                 </div>
                             </td>
                             <td t-if="din5008_document_information">
                                 <t t-out="din5008_document_information"/>
                             </td>
                         </tr>
+                        <tr/>
```

Comment:
Is it necessary ? (I know we talked about it live, but just so I don't forget :) )